instabot/bot/bot_follow.py

```diff
@@ -8,7 +8,7 @@ def follow(self, user_id):
     msg = ' ===> Going to follow `user_id`: {}.'.format(user_id)
     self.console_print(msg)
     if not self.check_user(user_id):
-        return True
+        return False
```

Comment:
Why did you make this change?

Reply:
Because, check user filters the user_id according to our criteria. If that user_id fits the target, function returns true and then bot should follow that user. Here, if user is not our target, follow function returns True and counts as followed, which is incorrect.